doc/site/content/docs/guides/getting-started/getting-started-with-rmlui.md

```diff
@@ -213,21 +214,26 @@ Here's a basic widget written by Mupersega.
 Let's take a look at different areas that are important to look at.
 
 `<div id="rml-starter-widget" class="relative" data-model="starter_model">`
-1. Here, we bind to the data model using `data-model`. This is what we will need to name the data model in our Lua script later. Everything inside the model will be in scope i nand beneath the div.
-2. Typically, it is reccommended to bind your data model inside of a div beneath `body` rather than `body` itself.
+1. Here, we bind to the data model using `data-model`. This is what we will need to name the data model in our Lua script later. Everything inside the model will be in scope and beneath the div.
+2. Typically, it is recommended to bind your data model inside of a div beneath `body` rather than `body` itself.
 
 ```html
 <div id="expanding-content" data-class-expanded="expanded">
     <input type="checkbox" value="expanded" data-checked="expanded"/>
 ```
-anything with `data-x` is a "data event". Talking about these is redundant, as they are documented [on the RmlUi docs site](https://mikke89.github.io/RmlUiDoc/pages/data_bindings/views_and_controllers.html). You can, however, piece together what's happening here:
-1. `data-class-expanded="expanded"` applies the `expanded` class to the div if the value `expanded` in the data model is `true`.
-2. `<input type="checkbox" value="expanded" data-checked="expanded"/>` This checkbox will set the data model value in `data-checked` to true.
-3. When the data model is changed, the document is rerendered automatically. So, the expanding div will have the `expanded` class applied to it or removed whenever the check box is toggled.
+any attribute starting with `data-` is a "data event". We will go through a couple below, but you can find out more here [on the RmlUi docs site](https://mikke89.github.io/RmlUiDoc/pages/data_bindings/views_and_controllers.html).
+1. Double curly braces are used to show values from the data model within the document text. e.g. `{{message}}` shows the value of `message` in the data model.
+2. `data-class-expanded="expanded"` applies the `expanded` class to the div if the value `expanded` in the data model is `true`.
+3. `<input type="checkbox" value="expanded" data-checked="expanded"/>` This checkbox will set the data model value in `data-checked` to true.
+4. When the data model is changed, the document is rerendered automatically. So, the expanding div will have the `expanded` class applied to it or removed whenever the check box is toggled.
+
+There are data bindings to allow you to loop through arrays (data-for) have conditional sections of the document (data-if) and many others. 
 
 ### The Lua
 
-Here is a
+Hint: For information on setting up your editor to provide intellisense behaviour see the [Lua Language Server guide]({{% ref "lua-language-server" %}}).
```

Comment:
```suggestion
> [!NOTE]
> For information on setting up your editor to provide intellisense behaviour see the [Lua Language Server guide]({{% ref "lua-language-server" %}}).
```

Also, I'm not sure this `ref` works, would you mind testing by having mise installed and running `mise run server` inside `doc/site`?

Reply:
> It does on mine

So you tested locally right? Resolving it

Reply:
Unresolving for the note thing, a bit opinionated styling but I think it makes sense. Feel free to resolve if you disagree (not a blocker)